.github/workflows/create_draft_release.yml

```
@@ -66,3 +66,15 @@ jobs:
             ${{steps.assets.outputs.artifacts_path}}/*.rpm
             ${{steps.assets.outputs.artifacts_path}}/*.tar.gz
             ${{steps.assets.outputs.artifacts_path}}/*.zip
+            ${{steps.assets.outputs.gitlab_artifacts_path}}/*.msi
+            ${{steps.assets.outputs.gitlab_artifacts_path}}/*.zip
+
+      - name: "Publish nuget packages to nuget.org"
+        working-directory: ${{steps.assets.outputs.artifacts_path}}
+        run: |
+          dotnet nuget push Datadog.Trace.{{steps.versions.outputs.full_version}}.nupkg --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+          dotnet nuget push Datadog.Trace.{{steps.versions.outputs.full_version}}.snupkg --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+          dotnet nuget push Datadog.OpenTracing.{{steps.versions.outputs.full_version}}.nupkg --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+          dotnet nuget push Datadog.OpenTracing.{{steps.versions.outputs.full_version}}.snupkg --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+          dotnet nuget push Datadog.Monitoring.Distribution.{{steps.versions.outputs.full_version}}-beta01.nupkg --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
+          dotnet nuget push dd-trace.{{steps.versions.outputs.full_version}}.nupkg --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
```

We can simplify this by globbing the nupkgs, and relying on the auto-push for snupkg. Have you already added the key to the repository?

```suggestion
          dotnet nuget push "*.{{steps.versions.outputs.full_version}}.nupkg" --api-key {{ secrets.NUGET_API_KEY }} --source https://api.nuget.org/v3/index.json
```

---

I chose to make this explicit by listing all nugets for more control. 
But thinking of it now, I can also change the api key to allow only new versions (i had allowed push new packages as well). So the first upload would need to be manual and here I can accept your change

---

Alternatively, if you prefer, we could make this a small Nuke target with a foreach loop to avoid the duplication, but if you're happy with the suggested change, I think that's unnecessary

---

Done in 1ad8ac42a336fecd7aca60a84fe6b98aca0109d7. Changed your suggestion to work with all packages, nupkg and snupkg. I hesitated to do 2 lines, but I figured doing it in one go may be better

---

> and relying on the auto-push for snupkg 

Oh just seen that. I didn't know this was a thing.

---

Yeah, I think you'll need to rely on it here, as there's no guarantee that the update you made will try and push the `nupkg` version before the `snupkg` version?

---

Done in 7ca30683cbcb69ae17fb5f2c4ab95d4e3e7cb6a3.
:TIL: Thank you